tests/default/indices/mapping.yaml

```diff
@@ -146,3 +146,40 @@ chapters:
       status: 200
       payload:
         acknowledged: true
+  - synopsis: Add new fields to index mapping.
+    path: /{index}/_mapping
+    method: POST
+    parameters:
+      index: movies
+    request:
+      payload:
+        properties:
+          genre:
+            type: text
+          rating:
+            type: float
+    response:
+      status: 200
+      payload:
+        acknowledged: true
+  - synopsis: Get specific field mappings for an index.
```

Comment:
Consider splitting tests for different paths into their own scenarios if there's no duplication, e.g. this one would be `tests/default/indices/mapping/field.yaml`.